fab/fab/services/templates/supervisor_celery_worker.conf

```diff
@@ -1,16 +1,16 @@
-[program:{{ project }}-{{ environment }}-celery_{{ celery_params.queue }}_{{ celery_params.worker_num }}]
+[program:{{ project }}-{{ environment }}-celery_{{ celery_params.comma_separated_queue_names }}_{{ celery_params.worker_num }}]
```

Comment:
probably worth deploying this once to staging to make sure supervisor program names can have commas

Reply:
Good call. I assumed it would, but you know what happens when you assume...

Reply:
This worked ✅